modules/pubmaticBidAdapter.js

```diff
@@ -217,6 +276,221 @@ function _checkParamDataType(key, value, datatype) {
   }
 }
 
+function _createNativeRequest(params) {
+  var nativeRequestObject = {
+    assets: []
+  };
+  for (var key in params) {
+    if (params.hasOwnProperty(key)) {
+      var assetObj = {};
+      if (!(nativeRequestObject.assets && nativeRequestObject.assets.length > 0 && nativeRequestObject.assets.hasOwnProperty(key))) {
+        switch (key) {
+          case 'title':
+            if (params[key].length) {
+              assetObj = {
+                id: NATIVE_ASSET_ID.TITLE,
+                required: params[key].required ? 1 : 0,
+                title: {
+                  len: params[key].length,
```

Comment:
it will be convenient to store the value of params[key] in a variable to make the code more readable. since we are using params[key] in almost everyline, the code size will reduce if we use a variable.